src/psyclone/tests/gocean1p0_test.py

```diff
@@ -1083,35 +1083,6 @@ def test_find_grid_access(monkeypatch):
     assert arg is None
 
 
```

Comment:
pylint says there are a lot of unused imports in this file.

Reply:
Done